### PR TITLE
Change tld to localhost

### PR DESCRIPTION
**Pull request by inverse**
As per discussion in #3526 change `.app` tld to `.localhost`.

Still need to change the default configuration in homestead. See: https://github.com/laravel/homestead/blob/7007b45c56a09d2afc8c1019854a2ea26b771ff0/src/Settings/HomesteadSettings.php#L103-L108

### CODE REVIEW

File: valet.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ Valet will automatically start its daemon each time your machine boots. There is
 
 By default, Valet serves your projects using the `.dev` TLD. If you'd like to use another domain, you can do so using the `valet domain tld-name` command.
 
-For example, if you'd like to use `.app` instead of `.dev`, run `valet domain app` and Valet will start serving your projects at `*.app` automatically.
+For example, if you'd like to use `.localhost` instead of `.dev`, run `valet domain app` and Valet will start serving your projects at `*.localhost` automatically.
```
Comment (brunogaspar) [inline valet.md]: Missed one here `valet domain app`.

But i think the example should be the other way around, but that's something for other topic.

Comment (inverse) [reply valet.md]: Thanks :+1:

**Comment by taylorotwell** [comment]
I don't really like `localhost`. Would prefer `dev` or something.

**Comment by drbyte** [comment]
I don't like `.localhost` either, but am warming up to `.test` which is safe.

**Pull request closed without merging**